SimulationAgent/SimulationRunner.cs

```diff
@@ -41,6 +41,9 @@ public class SimulationRunner : ISimulationRunner
         // Set of counter to optimize scheduling
         private readonly ConnectionLoopSettings connectionLoopSettings;
 
+        // Set of counters to optimize scheduling
```

Comment:
the comment (and the one on line 41) seems incorrect, these are not counters, are they?

Reply:
Good question, just following the convention of the other loop settings comments. I'll follow up in-person to better understand what should be happening and if we should change it for all of the loop settings members.